packages/application-generic/src/usecases/add-job/add-job.usecase.ts

```diff
@@ -110,9 +110,17 @@ export class AddJob {
       };
       options.attempts = this.queueService.DEFAULT_ATTEMPTS;
     }
+
+    const jobData = {
+      _environmentId: job._environmentId,
+      _id: job._id,
+      _organizationId: job._organizationId,
+      _userId: job._userId,
+    };
```

Comment:
Minimal information we need to store in BullMQ to be able to run the jobs. Unfortunately for certain dependencies with certain features we can't only store the JobId.